Development/Porting.md

```diff
@@ -80,9 +80,10 @@ Solution:
   external flashing use case, freeing and modifying ifd to reduce ME size and
   expend BIOS region in consequence of liberated space can be [found here](https://github.com/corna/me_cleaner/wiki/External-flashing#neutralize-and-shrink-intel-me-useful-only-for-coreboot)
 
-* Run `make BOARD=$BOARD_DIR` to setup the coreboot tree,
-using the new coreboot config file.  This will create the output directory
-`build/coreboot-4.8/newarch/`, and after a while, you should have `newarch.rom`
-in the top level directory.
+* Run `make BOARD=$BOARD_DIR` (where `$BOARD_DIR` is the board directory
+  (`x230`, `x230-flas`h and others found under `./boards` directory) to setup
```

Comment:
x230`, `x230-flas`h and others found under - > x230`, `x230-flash` and others found under

Reply:
NOTE TO SELF: stop making documentation edits right before bed or before the caffeine has taken effect in the morning.